src/test/modelmanager_test.cpp

```diff
@@ -640,6 +641,41 @@ TEST(ModelManager, HandlingInvalidLastVersion) {
     manager.reloadModelWithVersions(config);
     ASSERT_EQ(modelInstance1->getStatus().getState(), ovms::ModelVersionState::END);
     ASSERT_EQ(modelInstance2->getStatus().getState(), ovms::ModelVersionState::AVAILABLE);
+    ASSERT_EQ(modelInstance2->getStatus().getErrorCode(), ovms::ModelVersionStatusErrorCode::OK);
+}
+
+TEST(ModelManager, HandlingInvalidVersionAtBeginning) {
```

Comment:
I'm adding unit test for that behavior here:
https://github.com/openvinotoolkit/model_server/pull/726/files#diff-d6ce9ab93bcb5115bd96af8e9a5513155abe39ce8587a97e04b7099de7b62ff5R645

We should stick to one test.